packages/ra-ui-materialui/src/types.ts

```diff
@@ -32,7 +32,11 @@ export interface EditProps<
     redirect?: RedirectionSideEffect;
     resource?: string;
     transform?: TransformData;
-    title?: string | ReactElement;
+    title?:
+        | string
+        | ReactElement
+        | false
+        | ((arg0: RecordType | undefined) => string);
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏

```suggestion
        | ((record: RecordType | undefined) => string);
```